tests/test_config_entries.py

```diff
@@ -7433,7 +7464,10 @@ class _OptionsFlow(config_entries.OptionsFlow):
 
                 def __init__(self, entry) -> None:
                     """Test initialisation."""
-                    self.config_entry = entry
+                    with patch.object(frame, "_REPORTED_INTEGRATIONS", set()):
+                        self.config_entry = entry
+                    with patch.object(frame, "_REPORTED_INTEGRATIONS", set()):
```

Comment:
So the options setter would probably never report as the integration is already reported?

Reply:
If both the `config_entry` and the `options` get set, then it will only report once yes.

Reply:
The frame helper uses the line number as key
The mock always returns the same line number.
So in the real world it will log twice.

Reply:
That was what I was thinking but if the mock sets the line number I guess this makes sense for testing.
As you say, in the real world it should log twice.